### DISCUSSION TITLE
Issue using pvsm file for postprocessing

### DISCUSSION THREAD

**Discussion by jayjiejie** [byuflowlab/FLOWUnsteady#85] [Q&A]
Dear FLOWUnsteady users

I just started using the code and tired to go through the tutorials. I am having a bit of problem to use the paraview state file for visualise the tutorial data during postprocessing. For example, the [example/wing case](https://flow.byu.edu/FLOWUnsteady/examples/wing-4p2aoa/). I follow the instructions at the bottom of the page and downloaded the weber_particle02.pvsm file. When I load it into paraview, it failed and found some error with regards to not finding 'blowing_fdom.735.xmf', 'wing-weber2.vtk' etc. I think it may due to the different data names used when generating this pvsm file. 

Thank you very much.

**Comment by Turbulence1116** [comment]
Hi Shuang, we are on the same boat.

**Reply by Turbulence1116** [reply]
and I can't visualise the vortex particles on all my simulations, any idea?

**Comment by EdoAlvarezR** [comment]
The pvsm file in the ["Simple Wing" tutorial](https://flow.byu.edu/FLOWUnsteady/examples/wing-4p2aoa/) also visualizes the fluid domain shown at the top of the page, so it's a little confusing. I would recommend using the pvsm of the ["Tethered Wing" tutorials](https://flow.byu.edu/FLOWUnsteady/examples/tetheredwing/) instead if this is your first time using ParaView.

To help you guys out, I made a recording of me visualizing the example: [youtu.be/fjYz5SMBcM8](https://youtu.be/fjYz5SMBcM8)

**Reply by jayjiejie** [reply]
Hi Ed Alvarez

Thank you very much for helping us out.

Shuang
